code/datums/religions.dm

```diff
@@ -125,6 +128,12 @@
 		preacher.put_in_hands(holy_book)
 	religiousLeader = preacher.mind
 	convert(preacher, null)
+	if(keys[1]=="christianity")
+		message_admins("[key_name(preacher)] has selected Christianity and converted the entire crew.")
+		for(var/mob/living/carbon/human/H in player_list)
```

Comment:
even the cultists?

Reply:
I don't think being part of a religion has any interaction with cult at all, but I can change it for immersions if you like.

Reply:
Would instantly out any cultist, no?

Reply:
It would be difficult to out every cultist since the only time the mass conversion happens is the moment a chaplain joins. After that it'd be unreliable as to who simply immediately renounced.

That being said, you have a point. Maybe it's best to just convert everyone regardless. After all, religion has no interaction with cult at all so it doesn't really matter.